### PR TITLE
TM-100: Add prettier to BE

### PR DESCRIPTION
**Pull request by ManuelTS**
Closes TM-100. Uses pre-commit hook to execute eslint and prettier

### CODE REVIEW

File: be/README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ This are the commands available for docker and the build pipeline, you should ru
 
 Install:
 ```shell
-nvm use && npm i && npm run generate
+nvm use && npm i && npm run prepaire && npm run generate
```
Comment (ja-fra) [inline be/README.md]:
```suggestion
nvm use && npm i && npm run prepare && npm run generate
```

**Review by ja-fra — CHANGES_REQUESTED** [review]
Please do not use the pre-commit-hook - there's a intellij-plugin to enforce these rules.

**Comment by ManuelTS** [comment]
> 

Done, also written in README

**Comment by ManuelTS** [comment]
> Please do not use the pre-commit-hook - there's a intellij-plugin to enforce these rules.

Done, also written in README

File: be/.husky/pre-commit [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+cd be/ && npx lint-staged
```
Comment (ja-fra) [inline be/.husky/pre-commit]: No hooks, please

File: be/package.json [hunk 3]
```diff
@@ -8,13 +8,14 @@
...
-    "lint": "eslint ./src",
+    "lint": "eslint --cache --fix ./src",
     "build": "tsc && cp -r src/static ./dist",
     "generate": "rm -R -f src/generated/* && openapi --input ../openapi.yaml --output ./src/generated",
     "generate:build": "npm run generate && npm run build",
     "generate:build:start": "npm run generate && npm run build && npm run start",
     "build:start": "npm run build && npm run start",
-    "package": "npm run generate:build && copy-node-modules . ./dist"
+    "package": "npm run generate:build && copy-node-modules . ./dist",
+    "prepare": "cd ../ && husky install be/.husky"
```
Comment (ja-fra) [inline be/package.json]: Is husky required if you don't use git-hooks?

**Review by ja-fra — CHANGES_REQUESTED** [review]

File: be/.husky/pre-commit (hunk 2 above)
Comment (ManuelTS) [reply be/.husky/pre-commit]: Sorry I thought no hooks for prettier, I removed them all

File: be/package.json (hunk 3 above)
Comment (ManuelTS) [reply be/package.json]: No, removed

File: be/package.json [hunk 4]
```diff
@@ -8,13 +8,12 @@
...
     "debug": "DEBUG=express:* node dist/app.js",
     "test": "jest --passWithNoTests",
-    "lint": "eslint ./src",
+    "lint": "eslint --cache --fix ./src",
     "build": "tsc && cp -r src/static ./dist",
     "generate": "rm -R -f src/generated/* && openapi --input ../openapi.yaml --output ./src/generated",
     "generate:build": "npm run generate && npm run build",
     "generate:build:start": "npm run generate && npm run build && npm run start",
-    "build:start": "npm run build && npm run start",
-    "package": "npm run generate:build && copy-node-modules . ./dist"
```
Comment (ja-fra) [inline be/package.json]: Please do not delete the `package`-script

**Review by ja-fra — CHANGES_REQUESTED** [review]

File: be/package.json (hunk 4 above)
Comment (ManuelTS) [reply be/package.json]: Restored

**Review by ja-fra — APPROVED** [review]

**Pull request merged**